schemas/threats-schema.json

```diff
@@ -109,36 +102,30 @@
                 "entries": {
                   "type": "array",
                   "items": {
-                    "type": "object",
-                    "properties": {
-                      "reference-id": {
-                        "type": "string",
-                        "description": "Mapping identifier (e.g., MITRE technique ID)"
-                      },
-                      "strength": {
-                        "type": "integer"
-                      },
-                      "remarks": {
-                        "type": "string"
-                      }
-                    },
-                    "required": ["reference-id"],
-                    "additionalProperties": false
+                    "type": "string",
```

Comment:
Are the schema changes intentional? (changing the type from object to string here and renaming `entries` to `identifiers` on line 113)